docs/core/development/policy/release-process.rst

```diff
@@ -388,6 +388,7 @@ Open questions
 
 - What bugs should be considered release blockers?
 
+  - Generally, fixes for regressions since the last release in the series and difficult-to-mitigate problems with the release toolchain
```

Comment:
It is strange that we start with "generally" :)

And why a "fix" is a blocker. I was expecting that the bug is the blocker.

I don't understand what "difficult-to-mitigate problems" are.

Can you give an example?